index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Porfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="./Images/all.min.css">
+</head>
+<body>
+  <header class="container">
+    <a href="#" class="welcome">My Logo</a>
+    <button type="button"><img class="icon" src="./eclips/menuicon.png" alt="icon"></button>
```

Comment:
- [ ] Please wrap your hamburger icon inside a `nav` tag instead of a button to increase your page accessibility 👍 .

Reply:
okay thanks